gems/pending/appliance_console/database_replication.rb

```diff
@@ -0,0 +1,116 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'English'
+
+module ApplianceConsole
+  class DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REPMGR_CONFIG = '/etc/repmgr.conf'.freeze
+
+    attr_accessor :cluster_name, :node_number,
+                  :primary_database, :primary_database_username, :primary_database_password,
```

Comment:
The database name, username, and password will always be the same on both primary and standby so it might actually be more clear without the "primary" qualification. So `database_name`, `database_user`, `database_password` is probably the way to go here.